cli/cmd/integration_ctr_reg_limits.go

```diff
@@ -0,0 +1,107 @@
+//
+// Author:: Salim Afiune Maya (<afiune@lacework.net>)
+// Copyright:: Copyright 2021, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+)
+
+func castStringToLimitByLabel(labels string) []map[string]string {
+	out := make([]map[string]string, 0)
+
+	for _, label := range strings.Split(labels, "\n") {
+		kv := strings.Split(label, ":")
+		if len(kv) != 2 {
+			cli.Log.Warnw("malformed limit_by_label entry, ignoring",
+				"label", label,
+				"expected_format", "key:value",
+			)
+			continue
+		}
+		out = append(out, map[string]string{kv[0]: kv[1]})
+	}
+
+	return out
+}
+
+func askForV2Limits(answers interface{}) error {
+	custom := false
+	if err := survey.AskOne(&survey.Confirm{
+		Message: "Configure limit of scans by tags?",
+	}, &custom); err != nil {
+		return err
+	}
+
+	if custom {
```

Comment:
Could these questions be split into separate functions? Would be more readable to have

askForLimitScansTags()
askForLimitScansLabels()
...